config/config.sample.yaml

```diff
@@ -25,11 +25,34 @@ media:
   enable_profile_images: true # Enable download of profile images
 
 logging:
-  file: logs/twitter-bridge.log #Filepath of the current log file. (old ones are rotated.)
-  size: 50m # Maximum size (m,k,b) of a log file before it is rotated.
-  count: 3 # Maximum number of files to keep before old files are deleted.
+  file: logs/twitter-bridge.log #Filepath of the log file.
+  console: true
   compress: false # Compress the files in GZip format?
   level: verbose # One of error,warn,info,verbose,silly,silent
+  rotate:
+    daily: true # Should the log file be rotated each day.
+    size: 50m # Maximum size of a log file before it's rotated.
+    count: 3 # How many log files should be kept
+      # yy: Last two digits of the year.
+      # yyyy: Full year.
+      # M: The month.
+      # MM: The zero padded month.
+      # d: The day.
+      # dd: The zero padded day.
+      # H: The hour.
+      # HH: The zero padded hour.
+      # m: The minute.
+      # mm: The zero padded minute.
+      # ddd: The weekday (Mon, Tue, ..., Sun).
+    datePattern: "yyyy-MM-dd" # Date pattern to be used with daily.
```

Comment:
I would suggest hard-coding this instead of leaving it in config for one reason: for some ungodly reason, winston applies semantics to `datePattern` to determine what granularity of rotation to have. Which means you could end up with a log that _doesn't_ rotate daily despite the name of the thing.